.travis.yml

```diff
@@ -16,20 +16,24 @@ solution: Dlr.sln
 sudo: required
 dist: trusty
 
-dotnet: 2.0.0-preview2-006497
+dotnet: 2.0.0
 
 script:
   - dotnet restore Dlr.sln --packages ./packages
   - msbuild Build.proj /t:Build "/p:Mono=true;BuildFlavour=$CONFIGURATION"
   - pushd bin/$CONFIGURATION
   - mono ./net40/TestInternalDLR.exe
   - mono ./net45/TestInternalDLR.exe
+  - mono ./netcoreapp2.0/TestInternalDLR.exe
```

Comment:
.NET Core doesn't produce an exe (and in any case we shouldn't be testing the .NET Core stuff with Mono). You could try "dotnet ./netcoreapp2.0/TestInternalDLR.dll".